### PR TITLE
add org POST endpoint

### LINKED ISSUE #415: pg transition: Add organisation POST endpoint
Need to check if the orgs.add query is already there, if not:
- build it, 
- test it, 
- integrate it into the application, 
- add tests for validation

### PR DESCRIPTION
**Pull request by JMurphyWeb**
Add organisation POST endpoint.

closes #415

### CODE REVIEW

**Comment by mk4111** [comment]
@JMurphyWeb could you update the description with a link to the function (which is unmerged) that the tests rely on please? Thank you!

**Comment by JMurphyWeb** [comment]
@SimonLab ready for your viewing

**Comment by SimonLab** [comment]
@JMurphyWeb please reassign when the conflicts are resolved

**Comment by JMurphyWeb** [comment]
@SimonLab done 😄

File: server/handlers/orgs/add.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var Hoek = require('hoek');
+var helpers = require('../helpers.js');
```
Comment (SimonLab) [inline server/handlers/orgs/add.js]: I don't think "helpers" is used here, goodparts might have missed it.

File: test/organisations/add-org.test.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        t.equal(res.statusCode, 401, 'org is not added, admin is warned');
+        t.ok(res.payload.indexOf('name is not allowed to be empty'), 'Admin is told which field failed validation');
+
+        t.end();
+        pool.end();
+        server.stop();
+      });
+    });
+  });
+});
```
Comment (SimonLab) [inline test/organisations/add-org.test.js]: We could add another test which test when the logo_url is empty, I think the validation with Joi tell that this field is optional so it would be good to see that the query doesn't break.

Comment (JMurphyWeb) [inline test/organisations/add-org.test.js]: That should be tested in the plugin no?

Here my main aim was to test the overall flow, permissions, view logic and the validation.

Comment (JMurphyWeb) [inline test/organisations/add-org.test.js]: This is a bad test as it doesn't really test the org was added. At the time of writing I don't think i had the functionality to check.

**Comment by JMurphyWeb** [comment]
Removed and test added.

File: test/organisations/add-org.test.js (hunk 2 above)
Comment (SimonLab) [inline test/organisations/add-org.test.js]: Yes we will update add more tests to make the app stronger but let's finish the endpoints first
